### PR TITLE
Nazar/teature/settings view v1

### PR DESCRIPTION
**Pull request by Nazar-kob**

### CODE REVIEW

File: frontend/src/pages/Home/components/SettingsView/components/LeftPanel/LeftPanel.tsx [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import style from "./LeftPanel.module.css";
+
+type Props = {};
```
Comment (Taraslysak) [inline frontend/src/pages/Home/components/SettingsView/components/LeftPanel/LeftPanel.tsx:3]: Redundant type

File: frontend/src/pages/Home/components/SettingsView/components/RightPanel/RightPanel.module.css [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+}
+
+
+.copyIconContent {
+    width: 100%;
+    height: 100%;
+}
+
+.copyIconContent:hover {
+    color: rgb(116, 115, 113);
```
Comment (Taraslysak) [inline frontend/src/pages/Home/components/SettingsView/components/RightPanel/RightPanel.module.css:49]: why not hex value?

**Review by Taraslysak — APPROVED** [review]
Good job!

File: frontend/src/pages/Home/components/SettingsView/components/LeftPanel/LeftPanel.tsx (hunk 1 above)
Comment (Nazar-kob) [reply frontend/src/pages/Home/components/SettingsView/components/LeftPanel/LeftPanel.tsx:3]: this component will be changed in the future, so I did not remove it

File: frontend/src/pages/Home/components/SettingsView/components/RightPanel/RightPanel.module.css (hunk 2 above)
Comment (Nazar-kob) [reply frontend/src/pages/Home/components/SettingsView/components/RightPanel/RightPanel.module.css:49]: Ok, I fix

File: backend/app/router/business.py [hunk 3]
```diff
@@ -12,16 +12,11 @@
...
-    db: Session = Depends(get_db), current_user: m.User = Depends(get_current_user)
+    business: m.User = Depends(get_business_from_cur_user),
 ):
-    log(log.INFO, "get_business_cur_user [%s]", current_user)
-    business: m.Business = (
-        db.query(m.Business).filter_by(user_id=current_user.id).first()
-    )
-
-    check_access_to_business(business=business, data_mes=current_user)
+    log(log.INFO, "get_business_cur_user business_id:[%s]", business.id)
```
Comment (kipod) [inline backend/app/router/business.py:19]: %d

File: backend/app/router/business.py [hunk 4]
```diff
@@ -222,7 +217,9 @@ def get_order(business_uid: str, order_uid: str, db: Session = Depends(get_db)):
 
 
 @router.get(
-    "/{business_uid}", response_model=s.BusinessOut, status_code=status.HTTP_200_OK
+    "/{business_uid}",
+    response_model=s.BusinessOut,
+    status_code=status.HTTP_200_OK,
 )
 def get_business_out_by_uid(business_uid: str, db: Session = Depends(get_db)):
     log(log.INFO, "get_business_out_by_uid")
```
Comment (kipod) [inline backend/app/router/business.py:225]:
```python
log(log.INFO, "get_business_out_by_uid [%s]", business_uid)

File: frontend/.env.development [hunk 5]
```diff
@@ -1 +1 @@
-VITE_API_BASE_URL="https://api.fish-market.simple2b.net"
\ No newline at end of file
+VITE_API_BASE_URL="http://127.0.0.1:8008"
```
Comment (kipod) [inline frontend/.env.development:1]: EOL in EOF

**Review by kipod — APPROVED** [review]
Well done

**Pull request merged**